modules/quick-start-content-guidelines.adoc

```diff
@@ -0,0 +1,166 @@
+----
+
+* Avoid directional language, like In the top-right corner, click the icon. Directional language becomes outdated every time UI layouts change. Also, a direction for desktop users might not be accurate for users with a different screen size. Instead, identify something using its name.
+** *Correct example*:
++
+----
+In the navigation menu, click Settings.
+----
+** *Incorrect example*:
++
+----
+In the left-hand menu, click Settings.
+----
+* Do not identify items by color alone, like "Click the gray circle". Color identifiers are not useful for sight-limited users, especially colorblind users. Instead, identify an item using its name or copy, like button copy.
+** *Correct example*:
++
+----
+The success message indicates a connection.
+----
+** *Incorrect example*:
++
+----
+The message with a green icon indicates a connection.
+----
+
+* Use the second-person point of view, you, consistently:
+** *Correct example*:
++
+----
+Set up your environment.
+----
+** *Incorrect example*:
++
+----
+Let’s set up our environment.
+----
+
+[id="quick-start-content-guidelines-check-your-work-module_{context}"]
+== Check your work module
+
+* Once a user completes a step, a *Check your work* module appears. This module prompts the user to answer a yes or no question about the step results, which gives them the opportunity to review their work. For this module, you only need to write a single yes or no question.
+** If the user answers Yes, a check mark will appear.
+** If the user answers No, an error message appears with a link to relevant documentation, if necessary. The user then has the opportunity to go back and try again.
+
+[id="quick-start-content-guidelines-formatting-UI-elements_{context}"]
+== Formatting UI elements
+
+Format UI elements using these guidelines:
+
+* Copy for buttons, dropdowns, tabs, fields, and other UI controls: Write the copy as it appears in the UI and bold it.
+* All other UI elements—including page, window, and panel names: Write the copy as it appears in the UI and bold it.
+* Code or user-entered text: Use monospaced font.
+* Hints: If a hint to a navigation or masthead element is included, style the text as you would a link.
+* CLI commands: Use monospaced font.
+* In running text, use a bold, monospaced font for a command.
+* If a parameter or option is a variable value, use an italic monospaced font.
+* Use a bold, monospaced font for the parameter and a monospaced font for the option.
+
+.Additional resources
+
+* For voice and tone requirements, refer to link:https://www.patternfly.org/v4/ux-writing/brand-voice-and-tone[PatternFly’s brand voice and tone guidelines].
+* For other UX content guidance, refer to all areas of link:https://www.patternfly.org/v4/ux-writing/about[PatternFly’s UX writing style guide].
+* For any UX content guidance that is not covered in PatternFly, refer to link:https://www.carbondesignsystem.com/guidelines/content/guidance[IBM Carbon’s UX content guidelines].
```

Comment:
`https://www.carbondesignsystem.com/guidelines/content/guidance` is linking to non-existing page, it returns 404

Reply:
Thank you! Removing it